### PR TITLE
3.4 Backports

### PR DESCRIPTION
**Pull request by nyalldawson**

### CODE REVIEW

File: src/ui/qgsexpressionselectiondialogbase.ui [hunk 1]
```diff
@@ -85,22 +85,22 @@
   </layout>
   <action name="mActionSelect">
    <property name="text">
-    <string>Select features</string>
```
Comment (m-kuhn) [inline src/ui/qgsexpressionselectiondialogbase.ui]: Can you update the i18n folder too? This will break translations.

Comment (m-kuhn) [reply src/ui/qgsexpressionselectiondialogbase.ui]: If you are sure that it's picked up like that by lrelease all good for me.

Comment (nyalldawson) [reply src/ui/qgsexpressionselectiondialogbase.ui]: I'm far from sure -- I have zero experience with translations

Comment (m-kuhn) [reply src/ui/qgsexpressionselectiondialogbase.ui]: A simple `sed` over the source strings in `i18n/*.ts` might bring us to the safe side here. If not, I'd rather have translation work and accept imperfect capitalization in the English version.

Comment (nyalldawson) [reply src/ui/qgsexpressionselectiondialogbase.ui]: Oops I just realised I misread where your comment was - I thought it was on the select by expression change

Comment (nyalldawson) [reply src/ui/qgsexpressionselectiondialogbase.ui]: Wait - it is. Ignore me

Comment (nyalldawson) [reply src/ui/qgsexpressionselectiondialogbase.ui]: So I do see all these strings in i18n/*.ts : e.g.

``qgis_hi.ts:        <source>Filter Current Selection</source>```

Does that mean it's safe to backport?

Comment (m-kuhn) [reply src/ui/qgsexpressionselectiondialogbase.ui]: I found it in `qgis_de.ts` (without capitalization, obviously)

Comment (nyalldawson) [reply src/ui/qgsexpressionselectiondialogbase.ui]: Alright. I'll drop.

**Comment by nyalldawson** [comment]
@m-kuhn are you ok with this version of the PR?

**Comment by m-kuhn** [comment]
Yes. Thanks!
Anyway we should find a way to deal with this kind of translation changes in release branches for the future.

**Pull request merged**